lib/cylc/network/daemon.py

```diff
@@ -19,5 +19,5 @@
 
 from cylc.network.method import METHOD
 
-if METHOD == "https":
+if METHOD == "https" or "http":
```

Comment:
Should be `if METHOD == "https" or METHOD == "http":` or `if METHOD in ["https", "http"]:`